samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,392 @@
+   Currently, it is not needed for the UART version.
+
+If you want to make modifications to how the DFU is executed in this sample, you can do so by editing the manifest templates.
+See the :ref:`ug_nrf54h20_suit_customize_dfu` user guide for instructions and examples.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/suit/smp_transfer`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+Building and programming using the command line
+===============================================
+
+To build and program the sample to the nRF54H20 DK, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Over Bluetooth Low Energy
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample using the following ``west`` command, with the following Kconfig options set:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+
+         .. note::
+
+            If you are compiling in Windows and the build is unsuccessful due to the maximum path length limitation, use the following command:
+
+            .. code-block:: console
+
+               west build -p -b nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir --sysbuild -- -DFILE_SUFFIX=bt -DCONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+
+         If you want to further configure your sample, see :ref:`configure_application` for additional information.
+
+         After running the ``west build`` command, the output build files can be found in the :file:`build/DFU` directory.
+         The following build artifacts are found with both firmware binaries embedded as integrated payloads:
+
+          * :file:`root.suit` - This file is the most important envelope, containing the embedded :file:`application.suit` and :file:`radio.suit` files as integrated dependencies.
+            These files are used to execute DFU.
+          * :file:`application.suit` - This file is the envelope for the Application Core, containing embedded Application Core firmware as an integrated payload.
+          * :file:`radio.suit` - This file is the envelope for the Radio Core, containing embedded radio core firmware as an integrated payload.
+
+         For more information about files generated as output of the build process, see :ref:`app_build_output_files`.
+         For more information on the contents of the build directory, see :ref:`zephyr:build-directory-contents` in the Zephyr documentation.
+         For more information on the directory contents and structure provided by sysbuild, see :ref:`zephyr:sysbuild` in the Zephyr documentation.
+
+      #. Connect the DK to your computer using a USB cable.
+      #. Power on the DK.
+      #. Program the sample to the kit using the following command:
+
+         .. code-block:: console
+
+            west flash
```

Comment:
This is the standard command, so I would rather link to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/programming.html . Also, the recommended method for programming is using ``--erase``, so if this is an exception, it should be stated.